packages/block-library/src/post-time-to-read/deprecated.js

```diff
@@ -0,0 +1,27 @@
+const v1 = {
```

Comment:
The `displayAsRange` option itself is being added for the first time by this PR, so the block deprecation is unnecessary, right?